### PR TITLE
[Communication] - Phone Numbers - Extracting phone numbers from administration package

### PR DESCRIPTION
**Pull request by minnieliu**

### CODE REVIEW

File: sdk/communication/azure-communication-administration/README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Azure Communication Administration client library for Java
 
-The administration package provides capabilities for Phone Number Administration.
+The administration package previously provided capabilities for User and Token Management and for Phone Number Administration.
 
-Acquired phone numbers can come with many capabilities, depending on the country, number type and phone plan. Examples of capabilities are SMS inbound and outbound usage, PSTN inbound and outbound usage. Phone numbers can also be assigned to a bot via a webhook URL.
+Now, User and Token Management capabilities have been moved to `azure-communication-identity`.
+Phone Number Administration have been moved to `azure-communication-phonenumbers`.
```
Comment (DominikMe) [inline sdk/communication/azure-communication-administration/README.md]: nit
```suggestion
Phone Number Administration has been moved to `azure-communication-phonenumbers`.
```

File: sdk/communication/azure-communication-phonenumbers/README.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+  <artifactId>azure-communication-phonenumbers</artifactId>
+  <version>1.0.0-beta.4</version>
+</dependency>
+```
+
+## Key concepts
+
+
+### Initializing Phone Number Client
+The PhoneNumberClientBuilder is enabled to use Azure Active Directory Authentication
```
Comment (DominikMe) [inline sdk/communication/azure-communication-phonenumbers/README.md:33]: Looking forward to have this updated with the new `PhoneNumbersClient` API soon and then compare with before :-)

File: sdk/communication/azure-communication-phonenumbers/README.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+[coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
+[coc_contact]: mailto:opencode@microsoft.com
+[product_docs]: https://docs.microsoft.com/azure/communication-services/
+[package]: https://search.maven.org/artifact/com.azure/azure-communication-administration
+[api_documentation]: https://aka.ms/java-docs
+[source]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/communication/azure-communication-administration/src
+
+
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java%2Fsdk%2Fcommunication%2Fazure-communication-administration%2FREADME.png)
```
Comment (DominikMe) [inline sdk/communication/azure-communication-phonenumbers/README.md]:
```suggestion
![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java%2Fsdk%2Fcommunication%2Fazure-communication-phonenumbers%2FREADME.png)
```

File: sdk/communication/azure-communication-phonenumbers/README.md [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+
+Check out other client libraries for Azure communication service
+
+<!-- LINKS -->
+[cla]: https://cla.microsoft.com
+[coc]: https://opensource.microsoft.com/codeofconduct/
+[coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
+[coc_contact]: mailto:opencode@microsoft.com
+[product_docs]: https://docs.microsoft.com/azure/communication-services/
+[package]: https://search.maven.org/artifact/com.azure/azure-communication-administration
```
Comment (DominikMe) [inline sdk/communication/azure-communication-phonenumbers/README.md]:
```suggestion
[package]: https://search.maven.org/artifact/com.azure/azure-communication-phonenumbers
```

File: sdk/communication/azure-communication-phonenumbers/README.md [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+[api_documentation]: https://aka.ms/java-docs
+[source]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/communication/azure-communication-administration/src
```
Comment (DominikMe) [inline sdk/communication/azure-communication-phonenumbers/README.md]:
```suggestion
[source]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/communication/azure-communication-phonenumbers/src
```

File: sdk/communication/azure-communication-phonenumbers/pom.xml [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-communication-phonenumbers</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0.0-beta.4</version> <!-- {x-version-update;com.azure:azure-communication-phonenumbers;current} -->
+
+  <name>Microsoft Azure client phone numbers library for communication</name>
+  <description>
+    This package contains APIs for managing phone numbers in Microsoft Azure Communication Services.
+    For this release, see notes - https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication/azure-communication-administration/README.md and https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication/azure-communication-administration/CHANGELOG.md.
```
Comment (DominikMe) [inline sdk/communication/azure-communication-phonenumbers/pom.xml]:
```suggestion
    For this release, see notes - https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication/azure-communication-phonenumbers/README.md and https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication/azure-communication-phonenumbers/CHANGELOG.md.
```

File: sdk/communication/azure-communication-phonenumbers/pom.xml [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+
+  <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core</artifactId>
+      <version>1.13.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-communication-common</artifactId>
```
Comment (DominikMe) [inline sdk/communication/azure-communication-phonenumbers/pom.xml:64]: Do we still need the dependency for common?

File: sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersIntegrationTestBase.java [hunk 8]
```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-package com.azure.communication.administration;
+package com.azure.communication.phonenumbers;
 
 import java.time.OffsetDateTime;
 import java.util.ArrayList;
```
Comment (DominikMe) [inline sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersIntegrationTestBase.java:6]: We can probably delete the recordings below?

**Review by DominikMe — APPROVED** [review]
Thanks for extracting the new package, Minnie!

I found some more remaining "administration" strings :-) Otherwise lgtm ✅

File: sdk/communication/azure-communication-phonenumbers/pom.xml (hunk 7 above)
Comment (minnieliu) [reply sdk/communication/azure-communication-phonenumbers/pom.xml:64]: I believe so, we still need phoneNumberIdentifier

File: sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersIntegrationTestBase.java (hunk 8 above)
Comment (minnieliu) [reply sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersIntegrationTestBase.java:6]: Yea you're right, I'll delete the recordings

**Pull request merged**